source/assets/components/shoppingCard/shoppingCard.css

```diff
@@ -29,6 +29,10 @@ body {
   opacity: 0.9;
 }
 
+.input-group-text:hover {
```

Comment:
ooh, is this how you do hover effects?

Reply:
yep just append `:hover` to what you whatever CSS selector you want